modin/engines/dask/pandas_on_dask/frame/axis_partition.py

```diff
@@ -73,7 +73,7 @@ def deploy_axis_func(
         list
             A list of distributed.Future.
         """
-        client = get_client()
+        client = default_client()
```

Comment:
[`default_client()`](https://github.com/dask/distributed/blob/2c8684701eb3e19a48ef1ef7125ed4f9ccd9d682/distributed/client.py#L4487) returns the current active scheduler's client, in comparison with [`get_client()`](https://github.com/dask/distributed/blob/2c8684701eb3e19a48ef1ef7125ed4f9ccd9d682/distributed/worker.py#L3383), it does not create a new connection if there is no active one but raises an exception, which helps to [get an error as soon as possible](https://github.com/modin-project/modin/issues/3133#issuecomment-856698792) if something went wrong.

Reply:
Shouldn't we replace `get_client()` to `default_client()` in `modin/engines/dask/utils.py`?

Reply:
Yes, we should. Replaced.